packages/rum-core/src/domain/getSelectorFromElement.ts

```diff
@@ -144,6 +148,10 @@ function getPositionSelector(element: Element): string {
   return `${cssEscape(element.tagName)}:nth-of-type(${elementIndex})`
 }
 
+function getSelectorFromDetachedElement(element: Element, targetElementSelector: string) {
+  return combineSelector(`DETACHED>${cssEscape(element.tagName)}`, targetElementSelector)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Does this selector allow to select the node with `document.querySelector()`? If this is not the case, I'm wondering if we should just avoid computing the selector.

Reply:
Definitely things that we should discuss before enabling that everywhere.
I'd like to test it like that just to have a bit more info when it happens and having the prefix would help.